src/components/BooksList/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import Book from '../Book';
+
+function BooksList({ books }) {
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
```

Comment:
opps, we need to rearrange some file.
React bindings for Redux separate presentational components from container components. This approach can make your app easier to understand and allow you to more easily reuse components. In your components directory (presentational components) you have components that are not connected to the Redux store, and on your containers directory you have components that are connected to the Redux store. https://redux.js.org/basics/usage-with-react#presentational-and-container-components 

![image](https://user-images.githubusercontent.com/31897434/89564719-54646580-d815-11ea-9096-91fb7ce337ca.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/

Reply:
This article is too old and the writer wrote not to follow this in current practice